### DISCUSSION TITLE
Receiving user contributions with Studio v3

### DISCUSSION THREAD

**Discussion by msafi** [sanity-io/sanity#3407] [Sanity Studio Support]
Thank you for your work on v3, it seems very exciting!

I have a question about using the Studio to receive user-generated content.

We have our own system for managing user accounts and authentication. We would like to let our users submit content for approval. The obvious solution is that we build a UI for that. The UI would talk to an API end point to authenticate the request before saving it into our Sanity database. But before we build that UI I wanted to ask for advice here: can we leverage the existing Studio v3 components for that use-case so that we don't have to build our own UI from scratch?

**Comment by Glavin001** [comment]
@msafi did you find a solution for this? I have a similar use case.

**Reply by msafi** [reply]
@Glavin001 Ended up building components from scratch
